docs/application/web/get-started/web-service/first-service.md

```diff
@@ -0,0 +1,223 @@
+
+        ![Selecting the application type](media/create_project_wizard_app_Web.png)
+
+    4.  Select the **Basic UI Service** template and click **Next**.
+
+        ![Selecting the template](media/create_project_wizard_template_ma.png)
+
+    5.  Define the project properties and click **Finish**.
+
+        You can enter the project name (3-50 characters) and the unique package ID. You can also select the location and working sets by clicking **More properties**.
+
+        ![Defining properties](media/create_project_wizard_properties_ma.png)
+
+        The Project Wizard sets up configurations and creates the Web service files using the template. For more information on the Project Wizard and the available templates, see [Create Tizen Projects with Tizen Project Wizard](../../../tizen-studio/Web-tools/project-wizard.md).
+
+<a name="build"></a>
+## Build Web Service Package
+
+After the Web service project is created, you can implement the required features. In the following example, only the default features from the project template are used, thus no code changes are required.
+
+When your Web service code is ready, you must build the Web service. The building process performs a validation check and packs your resource files.
+
+Here we will build for signed package.
+
+To build the signed package, right-click the project in the **Project Explorer** view and select **Build Signed Project**.
+
+**Figure: Build Web Service**
+
+![Building the Web service](media/build_manual_a.png)
+
+After you have built the Web service, run it.
+
+<a name="run"></a>
+## Run Web Service
+
+You can run the Web service on the [emulator](../../tutorials/process/run-debug-app.md#emulator) or a [real target device](../../tutorials/process/run-debug-app.md#target).
+
+<a name="emulator"></a>
+### Run on Emulator
+
+To run the Web service on the emulator:
+
+1.  Launch an emulator instance in the [Emulator Manager](../../../tizen-studio/common-tools/emulator-manager.md):
+    1. In Tizen Studio menu, select **Tools \> Emulator Manager**.
+
+        ![Emulator Manager](media/emulator_icon.png)
+
+    2.  In the Emulator Manager, select a mobile emulator from the list and click **Launch**.
+
+        If no applicable emulator instance exists, [create a new one](../../../tizen-studio/common-tools/emulator-manager.md#create).
+
+        ![Launching the emulator](media/emulator_launch_m.png)
+
+        The emulator is launched in its own window. You can also see the new emulator instance and its folder structure in the **Device Manager**.
+
+        ![Emulator](media/emulator_window_m.png)
+
+2.  Generate a security profile.
+
+    Before you run the Web service, you must [sign your Web service package with a certificate profile](../../../tizen-studio/common-tools/certificate-registration.md) in Tizen Studio.
+
+3.  Run Web service:
+    
+    Web services run by Web applications. Therefore, you need to run a Web application firstly.
```

Comment:
IMO this can be a note, kindly share your thoughts. 
``suggestion 
> [!NOTE]  
> Web services require respective Web applications to run. Therefore, you need to run a Web application first.

Reply:
okay, keep it as it was.